src/main/java/com/dev/cinema/Main.java

```diff
@@ -44,11 +46,37 @@ public static void main(String[] args) {
         AuthenticationService authenticationService =
                 (AuthenticationService) injector
                         .getInstance(AuthenticationService.class);
-        authenticationService.register("SomeEmail@mail.com", "SomePassword");
+        User user = authenticationService.register("SomeEmail@mail.com", "SomePassword");
         try {
             authenticationService.login("SomeEmail@mail.com", "SomePassword");
         } catch (AuthenticationException e) {
             e.printStackTrace();
         }
+
+        ShoppingCartService shoppingCartService =
+                (ShoppingCartService) injector
+                        .getInstance(ShoppingCartService.class);
+        try {
+            shoppingCartService.getByUser(authenticationService
+                    .login("SomeEmail@mail.com",
+                            "SomePassword"));
+        } catch (AuthenticationException e) {
+            e.printStackTrace();
```

Comment:
I would not recommend to try-catch all of these. Save the result of `login()` to a variable and use it in `shoppingCarService`. But if you still want to keep it as it is - `e.printStackTrace()` is a bad practice. You can show the AuthenticationException message